src/mappings/DAOTracker/mapping.ts

```diff
@@ -0,0 +1,127 @@
+import { Address, ipfs, store } from '@graphprotocol/graph-ts';
+import {
+  BlacklistDAO,
+  DAOTracker,
+  ResetDAO,
+  TrackDAO,
+} from '../../types/DAOTracker/DAOTracker';
+import {
+  BlacklistedDAO,
+  DAOTrackerContract,
+  ResetDAO as ResetDAOEntity,
+} from '../../types/schema';
+import {
+  Avatar_0_0_1_rc_31,
```

Comment:
what is this postfix is for ? Avatar_0_0_1_rc_31 ?

Reply:
These names are currently auto generated [here](https://github.com/dOrgTech/subgraph/blob/ab7135abeb86ad27997b1d92acb7811ba8fc6ef9/ops/generate-subgraph.js#L137) based on the version stated [here](https://github.com/dOrgTech/subgraph/blob/ab7135abeb86ad27997b1d92acb7811ba8fc6ef9/src/mappings/DAOTracker/datasource.yaml#L7).

The reason is because datasource templates cannot have the same name, and we need a way of supporting multiple versions of the ABI.

Reply:
is that can be avoided somehow by getting the template from a pre defined mapping or something ?

Reply:
see https://github.com/daostack/subgraph/pull/380/files#r342098970 for a reference

Reply:
Responded to your comment linked above, will look into this.

Reply:
Looked into it, and responded to your comment here: https://github.com/daostack/subgraph/pull/380/files#r342098970